e2e/tests/test.ts

```diff
@@ -13,6 +14,9 @@ type AsyncReturnValue<T> = T | Promise<T>;
 const setupTasks: Promise<void>[] = [];
 const teardownTasks: Awaited<VoidFunction>[] = [];
 
+// makes possible call things in browser context.
+window.fabric = fabric;
```

Comment:
not going to do anything fancier than this. this works

Reply:
I am interestred where it is used